### PR TITLE
Scheduled dialog number of unsubscribed recipients

### PR DESCRIPTION
**Pull request by oleg-morozov**
JIRA: TNT-698

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by oleg-morozov** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7539/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/9400/ : SUCCESS in 11m 26s

File: libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/ScheduledMailDialogRenderer.tsx [hunk 1]
```diff
@@ -289,6 +284,10 @@ export class ScheduledMailDialogRendererUI extends React.PureComponent<
             startDate: new Date(schedule.when.startDate), // check time zones
             isValidScheduleEmailData: true,
             repeat: parseRepeatString(schedule.when.recurrence),
+            unsubscribedRecipients:
```
Comment (tomassatinsky) [inline libs/sdk-ui-dashboard/src/presentation/scheduledEmail/DefaultScheduledEmailDialog/ScheduledMailDialogRenderer/ScheduledMailDialogRenderer.tsx:287]: you don't need to take it from state - you can get it right from pros.editSchedule

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7544/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tests/196/ : SUCCESS in 11m 54s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-check-extended-sonar-PR2428

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7543/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1291/ : SUCCESS in 17m 09s

File: libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json [hunk 2]
```diff
@@ -312,6 +312,11 @@
         "comment": "Message of the scheduled email.",
         "limit": 0
     },
+    "dialogs.schedule.email.unsubscribed.recipients": {
+        "value": "{n, plural, one {1 recipient unsubscribed from this schedule} other {# recipients unsubscribed from this schedule}}.",
```
Comment (tomassatinsky) [inline libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json:316]: I would do `{n, plural, one {1 recipient unsubscribed} other {# recipients unsubscribed}} from this schedule.`

File: libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json [hunk 3]
```diff
@@ -312,6 +312,11 @@
...
+        "comment": "Translate only uppercase in '{n, plural, one {1 RECIPIENT UNSUBSCRIBED FROM THIS SCHEDULE} other {# RECIPIENTS UNSUBSCRIBED FROM THIS SCHEDULE}}.'",
```
Comment (tomassatinsky) [inline libs/sdk-ui-dashboard/src/presentation/localization/bundles/en-US.json:317]: This kind of comments is misleading, I think. I know we use it elsewhere. I would say just "translate with correct plural form for given language"

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1450/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7542/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1450/ : UNSTABLE in 24m 44s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7547/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/9404/ : SUCCESS in 10m 07s

**Review by tomassatinsky — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2428

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**